core/trino-parser/src/main/java/io/trino/sql/parser/AstBuilder.java

```diff
@@ -1555,30 +1558,89 @@ public Node visitDropRole(SqlBaseParser.DropRoleContext context)
                 visitIfPresent(context.catalog, Identifier.class));
     }
 
+    @Override
+    public Node visitGrantPrivileges(SqlBaseParser.GrantPrivilegesContext context)
+    {
+        List<String> privileges;
+        if (context.ALL() != null) {
+            privileges = ImmutableList.of();
```

Comment:
Why empty list? Just use `null` for this to remove any ambiguity about "empty means all" vs "empty means none"

Reply:
Changed.